test/integration/util/util.go

```diff
@@ -120,7 +122,7 @@ type OcRunner struct {
 }
 
 func NewOcRunner() *OcRunner {
-	p, _ := exec.LookPath("oc")
+	p := path.Join(os.Getenv(constants.MiniShiftHomeEnv), "cache", "oc", "v1.4.1", "oc")
```

Comment:
This is still hard code the oc version so I think will use `version.GetOpenShiftVersion()`